hadoop-hdds/interface-client/src/main/proto/hdds.proto

```diff
@@ -453,14 +453,14 @@ message ContainerBalancerConfigurationProto {
 }
 
 message DiskBalancerConfigurationProto {
-    optional double threshold = 1;
-    optional double diskBandwidth = 2;
+    optional string threshold = 1;
```

Comment:
@ChenSammi Currently ConfigurationReflectionUtil can not detect the type of "Double", so I revert the type back to string, please have a look.

Reply:
I see.  Do you think it's difficult to add a double support in ConfigurationReflectionUtil?

Reply:
Added "Double" to ConfigType in another PR, please have a look.
https://issues.apache.org/jira/browse/HDDS-7162